system/lib/standalone/standalone.c

```diff
@@ -123,17 +123,18 @@ double emscripten_get_now(void) {
   return (1000 * clock()) / (double)CLOCKS_PER_SEC;
 }
 
+#ifndef __USING_WASM_EXCEPTIONS__
 // C++ ABI
 
 // Emscripten disables exception catching by default, but not throwing. That
 // allows users to see a clear error if a throw happens, and 99% of the
 // overhead is in the catching, so this is a reasonable tradeoff.
 // For now, in a standalone build just terminate. TODO nice error message
-void
-__cxa_throw(void* ptr, void* type, void* destructor) {
+void __cxa_throw(void* ptr, void* type, void* destructor) {
```

Comment:
Another idea: Could these two just be declared as weak? Then we wouldn't need any build variation changes here at all I believe.

(Or, I guess splitting up into one file per function would achieve the same result, which you recently mentioned.)

Reply:
I thought about the weak thing too.. but I think we have to put libstandalonewasm on the command line first since it wants to override some stuff from libc (I think).  At least as of today if comes first on the command line.

Let me check the semantics of weak symbols that come first.. maybe its not an issue as I first thought.

Reply:
I was wrong ... weak will work fine.  Nice!